### PR TITLE
fix: Integration tests on windows

### LINKED ISSUE #1290: Fix integration tests on Windows
**Describe the bug**
> A clear and concise description of what the bug is.

Gradle clean build does not work on Windows, we should fix it

**To Reproduce**
> Steps to reproduce the behavior:

1. Run integration tests on Windows

**Additional context**
Please check issue #1246

### PR DESCRIPTION
**Pull request by adamfilipow92**
Fixes #1290 

Integration tests should work on windows.

### CODE REVIEW

**Review by piotradamczyk5 — APPROVED** [review]

**Review by piotradamczyk5 — APPROVED** [review]

File: test_runner/src/test/kotlin/ftl/run/status/MultiLinePrinterTest.kt [hunk 1]
```diff
@@ -19,6 +20,7 @@ class MultiLinePrinterTest {
 
     @Test
     fun test() {
+        if (isWindows) return
```
Comment (adamfilipow92) [inline test_runner/src/test/kotlin/ftl/run/status/MultiLinePrinterTest.kt:23]: I cannot get this test works even I downrange the ```JANSI``` version to 1.18

Comment (piotradamczyk5) [reply test_runner/src/test/kotlin/ftl/run/status/MultiLinePrinterTest.kt:23]: `assumeFalse(isWindows)`

**Comment by Sloox** [comment]
@adamfilipow92 @piotradamczyk5 if you need me to test this on a native windows machine, let me know.

**Comment by piotradamczyk5** [comment]
> @adamfilipow92 @piotradamczyk5 if you need me to test this on a native windows machine, let me know.

I think that @adamfilipow92 ❤️ Windows as well 😃

**Review by Sloox — APPROVED** [review]

**Pull request merged**